### PR TITLE
Quickstart for KOGITO-5289 Kogito DRL guide

### PR DESCRIPTION
**Pull request by lucamolteni**
https://github.com/quarkusio/quarkus/pull/19025

cc @mariofusco @tarilabs @jstastny-cz @danielezonca
**Check list**:

Your pull request:

- [x] targets the `development` branch
- [x] uses the `999-SNAPSHOT` version of Quarkus
- [x] has tests (`mvn clean test`)
- [x] works in native (`mvn clean package -Pnative`)
- [x] has native tests (`mvn clean verify -Pnative`)
- [x] makes sure the associated guide must not be updated
- [x] links the guide update pull request (if needed)
- [ ] updates or creates the `README.md` file (with build and run instructions)
- [x] for new quickstart, is located in the directory _component-quickstart_
- [x] for new quickstart, is added to the root `pom.xml` and `README.md`

### CODE REVIEW

**Comment by tarilabs** [comment]
I believe you need to un-flag `targets the development branch` in your description, @lucamolteni by looking at this PR.

**Review by jstastny-cz — COMMENTED** [review]
I think we're missing entry in root README.md as dictated by the PR template here.

File: kogito-drl-quickstart/pom.xml [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+  <artifactId>kogito-drl-quickstart</artifactId>
+  <version>1.0.0-SNAPSHOT</version>
+  <properties>
+    <compiler-plugin.version>3.8.1</compiler-plugin.version>
+    <maven.compiler.parameters>true</maven.compiler.parameters>
+    <maven.compiler.source>11</maven.compiler.source>
+    <maven.compiler.target>11</maven.compiler.target>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+    <quarkus.platform.artifact-id>quarkus-universe-bom</quarkus.platform.artifact-id>
```
Comment (jstastny-cz) [inline kogito-drl-quickstart/pom.xml:15]: quickstarts should use `quarkus-bom` as in other quickstarts I believe.

Comment (gsmet) [reply kogito-drl-quickstart/pom.xml]: I fixed it.

**Comment by gsmet** [comment]
I fixed a few minor things and force-pushed.

**Comment by gsmet** [comment]
Let's wait for CI and merge (things went well locally).

**Pull request merged**